doc/content/examples/index.md

```diff
@@ -0,0 +1,18 @@
+# Examples
+
+Inputs which demonstrate potential applications for TMAP-8 capabilities,
+along with walk-through guides which should in theory allow users to 
+leverage prior work.
+
+## Fuel cycle
+
+Because TMAP-8 is built on MOOSE, it brings MOOSE's capacity to solve ordinary differential
+equations using [ScalarKernels](syntax/ScalarKernels/index.md). These can be quite useful to
+model parts of the system at high levels of abstraction while working with detailed
+models of specific components. For an example, we re-create a fuel-cycle model using the 
+equations in [!cite](Abdou2021) to provide a high-level abstraction of a fuel-cycle in a potential 
+fusion power plant.
+
+[Fuel Cycle](examples/fuel_cycle/index.md)
```

Comment:
```suggestion
## [Fuel Cycle](examples/fuel_cycle/index.md)

Because TMAP8 is built on MOOSE, it brings MOOSE's capacity to solve ordinary differential
equations using [ScalarKernels](syntax/ScalarKernels/index.md). These can be quite useful to
model parts of the system at high levels of abstraction while working with detailed
models of specific components. For an example, we re-create a fuel cycle model using the 
equations in [!cite](Abdou2021) to provide a high-level abstraction of a fuel cycle in a potential 
fusion power plant.
```

Let's make "fuel cycle" a bit more consistent in this description, and move the link to the heading.